### ISSUE TITLE
STEAMERS Avatar And Info Not Fetched

### ISSUE THREAD

**Issue by Dr-Strange91** [dreamstalker/rehlds#673]
Hello, I'm using [DEV] Reunion 0.1.0.133. [Linux]

Actually the bug is Steamers avatars are not visible in score card [Press Tab] nor their info with, Shift + Tab is displayed, except of your personal avatar and those in your friend list + those whom you have sent friend request (Not sure about those who had sent you request).

So, it largely affects the server rush. Though server might be non-steam but still Steamers play a major part in your server.

I’m not sure if it is ReHLDS issue or Reunion, as, as far I remember, Same problem is with Stable Version Of Reunion, Too.

For More, You can See Screenshots And Other Details. See Here [Official ReHLDS Webs].

https://dev-cs.ru/threads/4174/page-2
http://www.dedicated-server.ru/vbb/showpost.php?p=303939&postcount=783

**Comment by Dr-Strange91** [comment]
So, I was going through different trials, in which I tried different versions of ReHLDS and Reunion, I tried versions from ReHLDS.605 and Reunion 0.175, removed ReAuth, ReVoice. I figured out after removing Reunion [Making Server Steam Only] fixed the problem, even with ReHLDS.605 and with latest both but as soon as I re Installed Reunion, Server failed to fetch Steam Players in Current Game [Shift + Tab] and no Avatar was also visible.

Changes were only taking place after restarting Server And Steam [Not CS].

I noticed new thing, as I made server Steam Only, the Avatar was visible and player who was with me starts appearing in Current Game And Recent Games.

Now, when I added Reunion again and restared server and Steam. The avatar and player from current disappeared, but as I opened recent [In which player was showing (in recent), because the time I made Server Steam, due to which The Player appeared in recent].

Now once Information fetched by steam from recent, The server started displaying avatar after it.

According to my conclusion, Reuinon is not able to fetch players from Steam and they are not shown in server and Current Game. Hence, No Avatar or other details are shown.

But In dproto such thing don't happen.
